src/Owin.Security.OpenIdConnect.Server/OpenIdConnectServerHandler.Endpoints.cs

```diff
@@ -1579,12 +1596,18 @@ internal partial class OpenIdConnectServerHandler : AuthenticationHandler<OpenId
 
         private async Task InvokeValidationEndpointAsync() {
             OpenIdConnectMessage request;
-
+            
+            // Note: The updated RFC standard says nothing about allowing GET requests beyond
+            // one mention that "...an authorization server offering token introspection MAY disallow
+            // the use of HTTP GET on the introspection endpoint and instead require the HTTP POST
+            // method to be used..." - previous versions of the standard explicitly mentioned the
+            // allowance of the HTTP GET method.
+            // See https://tools.ietf.org/html/rfc7662#section-2.1 and https://tools.ietf.org/html/rfc7662#section-4
             if (!string.Equals(Request.Method, "GET", StringComparison.OrdinalIgnoreCase) &&
                 !string.Equals(Request.Method, "POST", StringComparison.OrdinalIgnoreCase)) {
                 await SendErrorPayloadAsync(new OpenIdConnectMessage {
                     Error = OpenIdConnectConstants.Errors.InvalidRequest,
-                    ErrorDescription = "A malformed validation request has been received: " +
+                    ErrorDescription = "A malformed introspection request has been received: " +
```

Comment:
Right. Sorry. My brain is stuck on it being an "introspection" endpoint. I'll fix that.